doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -180,67 +214,95 @@ Deployment
 ~~~~~~~~~~
 
 - Run the ``SuperLink`` using |flowernext_superlink_link|_ before running, in sequence,
-  |flowernext_clientapp_link|_ (2x) and |flowernext_serverapp_link|_. There is no need
-  to execute `client.py` and `server.py` as Python scripts.
+  |flowernext_supernode_link|_ (2x).
 - Here's an example to start the server without HTTPS (only for prototyping):
 
 .. code-block:: bash
+    :emphasize-lines: 2,5,12
 
     # Start a Superlink
     $ flower-superlink --insecure
 
     # In a new terminal window, start a long-running SuperNode
-    $ flower-client-app client:app --insecure
+    $ flower-supernode \
+         --insecure \
+         --superlink 127.0.0.1:9092 \
+         --node-config "partition-id=0 num-partitions=2" \
```

Comment:
```suggestion
         --node-config "..." \
```

Reply:
I wouldn't use those `partition-id=0 num-partitions=2` since likely someone using `flower-supernode` doesn't have to think about partitions.

Reply:
True! Removed partitions and added ellipsis (...).